src/eei.cpp

```diff
@@ -81,6 +95,21 @@ Literal EthereumInterface::callImport(Import *import, LiteralList& arguments) {
       return Literal();
     }
 
+    if (import->base == Name("codeCopy")) {
+      cout << "codecopy\n";
+
+      uint32_t resultOffset = arguments[0].geti32();
+      uint32_t codeOffset = arguments[1].geti32();
+      uint32_t length = arguments[2].geti32();
+
+      vector<uint8_t> code(length);
```

Comment:
This is actually copying the currently executing code. Changing that because it requires a couple of changes.